src/app/layout.tsx

```diff
@@ -16,7 +17,10 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+      <Navbar/>
+        {children}
+        </body>
```

Comment:
```suggestion
      <body className={inter.className}>
        <Navbar/>
        {children}
      </body>
```

This is another very minor comment (just me being picky) but if you are making any other changes, can you indent that properly